upgrade/helpers/tasks.py

```diff
@@ -386,19 +455,29 @@ def sync_tools_repos_to_upgrade(client_os, hosts):
     tools_repo = entities.Repository(
         name=toolsrepo_name, product=tools_product, url=tools_repo_url,
         organization=org, content_type='yum').create()
+    logger.info("Entities product {} and repository {} is created successfully".
+                format(tools_product, toolsrepo_name))
     entities.Repository(id=tools_repo.id).sync()
+    logger.info("Entities repository sync operation has completed successfully for tool "
+                "repos name {}".format(tools_repo.name))
     cv.repository += [tools_repo]
     cv.update(['repository'])
+    logger.info("Content view publish operation is started successfully")
     call_entity_method_with_timeout(cv.read().publish, timeout=2500)
+    logger.info("Content view has published successfully within in 2500 seconds")
```

Comment:
```suggestion
    logger.info("Content view has published successfully")
```
You dont whether it really took 2500 seconds or 1000 seconds.